static/js/tools/statsvar_menu.tsx

```diff
@@ -158,49 +158,49 @@ class Node extends Component<NodePropType, NodeStateType> {
   }
 
   private canExpand() {
-    if (this.props.t !== "c") {
-      // a node can be expanded if it has >= 1 children
-      return this.childCnt(this.props.cd) !== 0;
-    } else {
-      // if the node is the top level node i.e. this.props.t === "c"
-      // we need to check if its child node has valid nodes.
-      let childCnt = 0; // number of valid child nodes
+    if (this.props.t === "p") {
+      // a property node can be expanded if it has >= 1 children
+      return this.hasChild(this.props.cd);
+    } else if (this.props.t === "c") {
+      // the top level node is expandable if has valid value node
+      // or valid property node
+      let hasChild = false;
```

Comment:
avoid have same name for variable and function "hasChild"

Reply:
Got it.